app/oauth.go

```diff
@@ -487,7 +492,7 @@ func (a *App) RevokeAccessToken(token string) *model.AppError {
 		return model.NewAppError("RevokeAccessToken", "api.oauth.revoke_access_token.del_token.app_error", nil, "", http.StatusInternalServerError)
 	}
 
-	if result := <-schan; result.Err != nil {
+	if result := <-schan; result != nil {
```

Comment:
`result` is very ambiguous. What do you thing about naming this just `err`?

Reply:
Make sense to me.